cloud/google/metadata.go

```diff
@@ -101,7 +101,10 @@ const masterEnvironmentVars = `
 KUBELET_VERSION={{ .Machine.Spec.Versions.Kubelet }}
 VERSION=v${KUBELET_VERSION}
 PORT=443
-MACHINE={{ .Machine.ObjectMeta.Name }}
+NAMESPACE={{ .Machine.ObjectMeta.Namespace }}
```

Comment:
Just a recommendation, but it seems easier to understand if this contatenation of the Namespace + "/ + Machine took place above in the go code. It would be in in masterMetadata(...) and nodeMetadata(...) and the value of machine would already contain the namespace and wouldn't need to be concatenated in this bash file.

Reply:
I probably should rename MACHINE as MACHINE_FULLPATH to reduce confusion. Machine object has name and namespace in its metadata. When we annotate the node object, we need to use namespace/name instead of just name if we want to support arbitrary namespace

Reply:
As this annotation logic is handled by scripts, it makes sense to include the logic here.